org.mixedrealitytoolkit.core/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
+## [3.2.1-development] - 2024-04-11
+
+### Fixed
+
+* Fixed missing [CanEditMultipleObject] attributes as per Bug 573 [PR #698](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/698)
+
 ## [3.2.0-development] - 2024-03-20
```

Comment:
Only the top version should have -development....

Please remove -development for older releases.  

(Same comment for all other MD changelogs)

Reply:
Done, please re-review whenever you have time.  Resolving this.